tasks.php

```diff
@@ -728,6 +739,8 @@ function handle_action_on_a_specified_task()
     }
     elseif ($action == 'add_comment') {
         if (!empty($_POST['task_comment'])) {
+            global $code_url;
```

Comment:
Put global declarations at the top of the function.

Reply:
Fixed in 8a53710d6c0fc4d8f798834b13a96575251b00b8, switched `$code_url` to `$tasks_url` thanks to @70ray in f49ee4e079d3571be2a841eee4cb9061b4c3a8be.